home/Gaming.md

```diff
@@ -50,8 +50,7 @@ You can use Proton to play your Windows games on Linux.
 
 CachyOS provides various Proton versions for improved performance, including `proton-cachyos`, `proton-ge-custom`, `proton-tkg-git`, and the official Proton versions `proton-experimental` and `proton`.
 
-Bottles
--------
+Additionally, [Protonup-qt](https://aur.archlinux.org/packages/protonup-qt) is a easy way to install and manage [Wine-GE](https://github.com/GloriousEggroll/wine-ge-custom) and [Proton-GE](https://github.com/gloriouseggroll/proton-ge-custom) installations depending on game launcher.
```

Comment:
We don't recommend to use protonup-qt in performance aspects.
Better point to the packages which are in the repo:
proton, proton-ge-custom, proton-cachyos, proton-experimental. These are compiled against systemlibs and should bring a more smooth experience.
 Same for wine-ge-custom :)

Reply:
It was mainly added as some people would prefer a GUI alternative, has been taken off with commit 7e2e5ce.